src/integration/java/io/pinecone/integration/dataPlane/UpsertAndQueryServerlessTest.java

```diff
@@ -74,7 +74,7 @@ public void upsertOptionalVectorsAndQueryIndexSyncTest() throws Exception {
         index.upsert(vectors, namespace);
 
         // wait sometime for the vectors to be upserted
-        Thread.sleep(5000);
+        Thread.sleep(7500);
```

Comment:
This and the change below was to give a bit more time for freshness, which can be an issue with serverless indexes.